force-app/main/default/prompts/OrderBuilder.prompt-meta.xml

```diff
@@ -0,0 +1,118 @@
+    </promptVersions>
+    <promptVersions>
+        <body
+        >You can drag products from the list into the basket. As you add products, the totals will update and relate the records to the order. &lt;br&gt;&lt;br&gt; The drop is handled using the standard &lt;a href=&quot;#=https://mzl.la/36N7i4c&quot;&gt;Drag &amp; Drop APIs&lt;/a&gt;.</body>
+        <dismissButtonLabel>Close</dismissButtonLabel>
+        <displayPosition>TopCenter</displayPosition>
+        <displayType>FloatingPanel</displayType>
+        <isPublished>false</isPublished>
+        <masterLabel>Order Builder Step 3</masterLabel>
+        <shouldDisplayActionButton>false</shouldDisplayActionButton>
+        <shouldIgnoreGlobalDelay>false</shouldIgnoreGlobalDelay>
+        <stepNumber>3</stepNumber>
+        <targetAppDeveloperName>EBikes</targetAppDeveloperName>
+        <targetPageKey1>Order__c</targetPageKey1>
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <title>Order</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >You can now modify the price &amp; quantity of the product being ordered. Once modified the totals on the order will be updated.&lt;br&gt;&lt;br&gt;This is done using &lt;a href=&quot;https://sfdc.co/B8HXD&quot;&gt;Custom Events&lt;/a&gt; sent up from the Order Item Component.</body>
+        <dismissButtonLabel>Close</dismissButtonLabel>
+        <displayPosition>TopCenter</displayPosition>
+        <displayType>FloatingPanel</displayType>
+        <isPublished>false</isPublished>
+        <masterLabel>Order Builder Step 4</masterLabel>
+        <shouldDisplayActionButton>false</shouldDisplayActionButton>
+        <shouldIgnoreGlobalDelay>false</shouldIgnoreGlobalDelay>
+        <stepNumber>4</stepNumber>
+        <targetAppDeveloperName>EBikes</targetAppDeveloperName>
+        <targetPageKey1>Order__c</targetPageKey1>
+        <targetPageKey2>view</targetPageKey2>
+        <targetPageType>standard__recordPage</targetPageType>
+        <title>Order Details</title>
+        <userAccess>Everyone</userAccess>
+        <userProfileAccess>Everyone</userProfileAccess>
+        <versionNumber>1</versionNumber>
+    </promptVersions>
+    <promptVersions>
+        <body
+        >&lt;p&gt;Now that you have seen Lightning Web Components used to create an interactive ordering experience, you can learn how to build them yourself! &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;As demonstrated, you can follow web standards like using Custom Events &amp;amp; The Drag &amp;amp; Drop API to communicate across components. &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Make sure you check out the &lt;b&gt;Property Explorer &lt;/b&gt;Tab for more great ways you can innovate with Lightning Web Components.&lt;/p&gt;</body>
```

Comment:
This is calling out the "Property Explorer", I guess that you mean "Product Explorer"